Templating/Helper/DateTimeHelper.php

```diff
@@ -140,12 +142,14 @@ public function format($datetime, $pattern, $locale = null, $timezone = null)
     }
 
     /**
+     * NEXT_MAJOR: Change to $date to \DateTimeInterface.
+     *
      * @param \IntlDateFormatter $formatter
-     * @param \Datetime          $date
+     * @param \DateTime          $date
      *
      * @return string
      */
-    public function process(\IntlDateFormatter $formatter, \Datetime $date)
+    public function process(\IntlDateFormatter $formatter, \DateTime $date)
```

Comment:
When, if you leave `\DateTime` as is, your PR will have no effect, right?

You should remove the typehint from method signature and do a manual check of the argument to be BC.

cc @greg0ire